unattended_installer/install_functions/installCommon.sh

```diff
@@ -309,6 +347,32 @@ function installCommon_installCheckDependencies() {
 
 }
 
+function installCommon_installChrome() {
+
+    dep="chrome"
+    common_logger "Installing ${dep}."
+
+    if [ "${sys_type}" == "yum" ]; then
+        chrome_package="/tmp/wazuh-install-files/chrome.rpm"
+        common_curl -so "${chrome_package}" https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm --max-time 100 --retry 5 --retry-delay 5 --fail
```

Comment:
Google does not maintain older versions of Google Chrome (or Chromium) officially. This is caused by the vulnerabilities reported frequently in these applications. Because of this, we would have to install an older version of Chrome extra-officially, and this version may have vulnerabilities fixed in newer versions. 

As a conclusion, it seems that the proposal is not a good idea. The update of the package should not cause the break of the PDF generation reports.

Reply:
All clear, thanks!